src/components/DataTable/DataTable.module.css

```
@@ -11,11 +11,177 @@
+
+/* ── Screen-reader only / announcer ───────────────────── */
+
+.srOnly,
+.srAnnounce {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)